examples/seq2seq/input_pipeline.py

```diff
@@ -15,62 +15,69 @@
 """Input pipeline for seq2seq addition example."""
 
 import random
+from typing import Any, Dict, Generator, Optional, Tuple
 
-import jax
 import jax.numpy as jnp
 import numpy as np
 
+Array = Any    # pylint: disable=invalid-name
```

Comment:
I agree I was a bit sloppy here, I've now changed it so that I use `np.ndarray` here for numpy arrays, and `Array` for jax numpy arrays. I don't think we use `jnp.ndarray` for `Array` in our codebase, we usually use `Any`.

IIRC @levskaya once said that `jnp.ndarray` is too restrictive in some cases, although I can't remember why exactly... (maybe he can enlighten us?)

Reply:
it has been tricky to get right in the past - we might want to check with chex/jax people what the best type-annotations are these days.